Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/ContentDialogSamplePage.xaml

```diff
@@ -12,14 +12,97 @@
 	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 		<local:SamplePageLayout>
 
+			<local:SamplePageLayout.MaterialTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<smtx:XamlDisplay UniqueKey="ContentDialogSamplePage_Material_Default"
+										  smtx:XamlDisplayExtensions.Header="Default ContentDialog">
+
+							<Button Content="Open dialog"
+									Click="Button_Click" />
+							<!-- with the following C# code
+private async void Button_Click(object sender, RoutedEventArgs e)
+{
+	var contentDialog = new ContentDialog()
+	{
+		Content = "Hello world!",
+		PrimaryButtonText = "OK",
+		XamlRoot = this.XamlRoot
+	};
+	await contentDialog.ShowAsync();
+}
+-->
+						</smtx:XamlDisplay>
+
+						<smtx:XamlDisplay UniqueKey="ContentDialogSamplePage_Material_Title"
+										  smtx:XamlDisplayExtensions.Header="ContentDialog with Title">
+
+							<Button Content="Open dialog"
+									Click="Button_Click2" />
+							<!-- with the following C# code
+private async void Button_Click2(object sender, RoutedEventArgs e)
+{
+	var contentDialog = new ContentDialog
+	{
+		Title = "Notice",
+		Content = "This is a very important message.",
+		PrimaryButtonText = "OK",
+		XamlRoot = this.XamlRoot
+	};
+	await contentDialog.ShowAsync();
+}
+-->
```

Comment:
Are we keeping these comments?

Reply:
IMO we should keep them as they conveniently show how to open the ContentDialog to the user: ![image](https://github.com/unoplatform/Uno.Gallery/assets/78549750/24aeabd2-ba22-4a54-a7a0-01b1ddcc9e3c)